src/components/app/pageBillDetails/partials/voteSection/dtsiAvatarBox.tsx

```diff
@@ -55,11 +55,12 @@ export function DTSIAvatarBox(props: DTSIAvatarBoxProps) {
     >
       <motion.div
         animate={{ opacity: 1, y: 0 }}
+        className="flex flex-col gap-2"
         initial={{ opacity: 0.33 }}
         key={person.slug}
         transition={{ duration: 0.75 }}
       >
-        <div className="relative mx-auto w-fit">
+        <div className="relative mx-auto w-fit" style={{ height: AVATAR_SIZE }}>
```

Comment:
np: Doesn't it work like that?
```suggestion
        <div className="relative mx-auto w-fit h-fit">
```